article-submission-example/.classpath

```diff
@@ -12,7 +12,7 @@
 			<attribute name="gradle_used_by_scope" value="main,test"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-18/"/>
```

Comment:
What's the significance of JDK version 18?  Is it the earliest version we can use with the Jakarta libraries?

Can we use the earliest supported LTS version (8, 11, 17 or 21)?

Reply:
Downgraded to Java11 that is the lowes LTS version supported by jakarta

Reply:
Any idea why we didn't need to update https://github.com/NorthConcepts/TemplateMaster/blob/master/.github/workflows/gradle.yml#L15 to JDK 11 for the build to work?

Is JDK 11 needed for runtime, but not compile-time??? 🤔

Reply:
Nothing is happening when we build the root project: [](https://github.com/NorthConcepts/TemplateMaster/actions/runs/8767555969/job/24060936390#step:4:30)
in the action you can see we are not running any task since we don't have a build.gradle for the root project.

If we were performing this on the library(or the other subprojects) then the compilation will fail(Tested it again in my local and it fails for jdk 1.8 stating that Jakarta was compiled by jdk 11).
Also the build.gradle will fail since the action uses gradle 4.7 and we added the content filter that is not supported by it.

Let me know if i should change anything.

Reply:
Sounds good.

Yes, please update the pipeline to run a simple compile -- like we do for DP examples -- to make sure there are no syntax or version issues.

Reply:
Done, I used a different approach that would not require us to add another build.gradle file